website/docs/openshift/microshift/index.md

```diff
@@ -0,0 +1,46 @@
+---
+sidebar_position: 104
+title: MicroShift
+description: Create a MicroShift cluster with Podman Desktop
+keywords: [podman desktop, podman, minc extension, microshift]
+tags: [install-minc-extension, create-a-microshift-cluster]
+---
+
+# Creating a MicroShift cluster
+
+With Podman Desktop, you can use the MicroShift in a container (MINC) extension to create a lightweight Kubernetes cluster that runs on minimal resources, such as CPU, RAM, and storage. The cluster is deployed as a container within your Podman environment. After setting up a MicroShift cluster, you can:
```

Comment:
I believe it should also say "create a lightweight OpenShift cluster"? Since it's technically a stripped version of OpenShift (MicroShift).